substratevm/src/com.oracle.svm.test/src/META-INF/native-image/com.oracle.svm.test/native-image.properties

```diff
@@ -1,7 +1 @@
-Args= \
-  --initialize-at-run-time=com.oracle.svm.test \
-  --initialize-at-build-time=com.oracle.svm.test.AbstractClassSerializationTest,com.oracle.svm.test.SerializationRegistrationTest \
-  --features=com.oracle.svm.test.SerializationRegistrationTestFeature \
-  -H:+AllowVMInspection \
-  --features=com.oracle.svm.test.AbstractServiceLoaderTest$TestFeature,com.oracle.svm.test.NoProviderConstructorServiceLoaderTest$TestFeature \
-  --add-exports=org.graalvm.nativeimage.builder/com.oracle.svm.core.containers=ALL-UNNAMED
+Args=--initialize-at-run-time=com.oracle.svm.test --initialize-at-build-time=com.oracle.svm.test.AbstractClassSerializationTest,com.oracle.svm.test.SerializationRegistrationTest,com.oracle.svm.test.LambdaClassSerializationTest,com.oracle.svm.test.LambdaClassDeserializationTest --features=com.oracle.svm.test.SerializationRegistrationTestFeature -H:+AllowVMInspection --features=com.oracle.svm.test.AbstractServiceLoaderTest$TestFeature,com.oracle.svm.test.NoProviderConstructorServiceLoaderTest$TestFeature --features=com.oracle.svm.test.SerializationRegistrationTestFeature -H:+AllowVMInspection --features=com.oracle.svm.test.AbstractServiceLoaderTest$TestFeature,com.oracle.svm.test.NoProviderConstructorServiceLoaderTest$TestFeature --add-exports=org.graalvm.nativeimage.builder/com.oracle.svm.core.containers=ALL-UNNAMED
```

Comment:
Seems like an unrelated chage?

Reply:
I actually added 2 classes for build time initialization.